package/system/procd/patches/0001-uxc-fix-compilation-error-caused-by-unused-variable.patch

```diff
@@ -0,0 +1,36 @@
+From 2ddf0005298e08ba1e358d95be6b826c56a7d1fc Mon Sep 17 00:00:00 2001
+From: Christian Marangi <ansuelsmth@gmail.com>
+Date: Tue, 29 Nov 2022 16:33:23 +0100
+Subject: [PATCH] uxc: fix compilation error caused by unused variable
+
+Fix compilation error caused by unused verbose variable.
+
+Signed-off-by: Christian Marangi <ansuelsmth@gmail.com>
```

Comment:
I'm thinking. This was fixed in the upstream repository in this commit https://git.openwrt.org/?p=project/procd.git;a=commit;h=0ee73b2c

Because the commit itself is huge, there are two options:
1) Do it as you did, even though, there should be some reference that it is applied in upstream
2) Consider creating openwrt-21.02/22.03 branch for procd and have this included in the upstream repository or updating it.

Reply:
Think we should add ref tag? A new branch seems overkill to just fix a verbose variable?

Any suggestion on how to correctly ref the upstream commit?

Reply:
@jow- I wonder what do you prefer for this. Is a patch acceptable?